src/Resources/fonts.js

```diff
@@ -0,0 +1,7 @@
+const DEFAULT_FONT = "'Source Sans Pro', -apple-system, Helvetica, sans-serif";
```

Comment:
I think that ```DEFAULT_FONT_FAMILY``` would be more descriptive naming for this constant, since there are multiple fonts, not just one.